bom/pom.xml

```diff
@@ -86,7 +86,9 @@
       <version.hibernate.search>5.8.1.Final</version.hibernate.search>
       <version.hikaricp>2.4.6</version.hikaricp>
       <version.java>1.8</version.java>
-      <version.javax.cache>1.0.0</version.javax.cache>
+      <version.javax.cache>1.1.0-SNAPSHOT</version.javax.cache>
+      <version.cdi>1.2</version.cdi>
+      <version.jboss.marshalling>2.0.0.Beta3</version.jboss.marshalling>
```

Comment:
Redefining JBoss Marshalling ? It is defined as 2.0.2.Final a couple of lines below